### PR TITLE
[feature](multi-catalog) support clickhouse jdbc catalog

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #14527

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
Hi, you need add a judgment in `org.apache.doris.external.jdbc.JdbcClient#parseDbType`

**Comment by zy-kkk** [comment]
> Hi, you need add a judgment in `org.apache.doris.external.jdbc.JdbcClient#parseDbType`

done

File: fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java [hunk 1]
```diff
@@ -486,6 +500,68 @@ public Type postgresqlTypeToDoris(JdbcFieldSchema fieldSchema) {
...
 
+    public Type clickhouseTypeToDoris(JdbcFieldSchema fieldSchema) {
+        String ckType = fieldSchema.getDataTypeName();
+        if (ckType.startsWith("LowCardinality")) {
+            ckType = ckType.substring(15, ckType.length() - 1);
+        }
+        if (ckType.startsWith("Nullable")) {
+            ckType = ckType.substring(9, ckType.length() - 1);
+        }
+        if (ckType.startsWith("Decimal")) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java:511]: Should be use `if else` for these conditions?

Comment (zy-kkk) [reply fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java:513]: done

**Review by BePPPower — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**